blob/src/main/java/com/bazaarvoice/emodb/blob/BlobStoreModule.java

```diff
@@ -175,6 +176,8 @@ protected void configure() {
 
         bind(StorageProvider.class).to(AstyanaxStorageProvider.class).asEagerSingleton();
         bind(MetadataProvider.class).to(AstyanaxStorageProvider.class).asEagerSingleton();
+        bind(S3StorageProvider.class).asEagerSingleton();
+        bind(MetadataProvider.class).to(AstyanaxStorageProvider.class).asEagerSingleton();
         bind(DataCopyDAO.class).to(AstyanaxStorageProvider.class).asEagerSingleton();
         bind(DataPurgeDAO.class).to(AstyanaxStorageProvider.class).asEagerSingleton();
```

Comment:
We need to implement a DataPurgeDAO and DataCopyDAO S3 so that table operations occur properly.

Reply:
sure, but it's out of scope of this ticket and will be done in scope of implementation of move table dropwizard admin task